src/input/controller.rs

```diff
@@ -0,0 +1,150 @@
+use super::{ControllerPort, Pad, WPad};
+use super::wpad::WPadButton;
+use super::pad::PadButton;
+
+pub enum ControllerType {
+    Gamecube,
+    Wii,
+}
+
+pub enum Button {
+    Left,
+    Right,
+    Up,
+    Down,
+    TrigL,
+    TrigR,
+    TrigZ,
+    TrigZL,
+    TrigZR,
```

Comment:
Any reason why it's TrigX instead of TriggerX?

(Sorry if these questions are dumb - I'm new to both Rust and Wii dev)

Reply:
This is also preference. Its easier to type then TriggerX or TriggerZL instead of TrigL or TrigZL

Reply:
Good point.